UnitTests/PaymentsCoreTests/NetworkingTests/APIClientTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+@testable import PaymentsCore
+
+class APIClientTests: XCTestCase {
```

Comment:
🤔  We now have 3 files for APIClientTests at different nest levels / with different names. We should clear this up.

Reply:
The testing structure got a little wonky after the networking PR. Also note, the file structure in XCode doesn't always accurately reflect the file structure in the repo, sometimes it gets out of sync and needs to be manually fixed.

![Screen Shot 2021-10-05 at 9 55 05 AM](https://user-images.githubusercontent.com/35243507/136047928-a76a50e9-5cf3-453f-bc54-05a459cf6c4d.png)

Reply:
I think for this PR, let's just consolidate the APIClient_Tests into one place. In a follow up PR, let's fix this structure.

Reply:
Ope this is everything I see.

![Screen Shot 2021-10-05 at 11 18 43 AM](https://user-images.githubusercontent.com/80476442/136052423-36fc67cf-41e6-446a-9a51-7170ee58a965.png)

Must've been my bad fixing merge conflicts in project file, I'll get this cleanup now :D

Reply:
You're looking in the XCode navigator ^

Reply:
all `APIClientTests` file has been removed :smile: